content/concepts/introduction/protocols/ping.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Ping"
+description: "The ping protocol is a simple request response protocol."
+weight: 20
+---
+
+## What is Ping?
+
+Ping is a network utility used to test a node's reachability.
+The ping protocol is a simple request-response protocol that measures the RTT
+for requests sent from an originating node to a destination node by echoing
+a request payload.
+
+## Ping in LibP2P
+
+In LibP2P, a peer opens a stream, sends a request with a payload of 32 random bytes,
+and the destination peer responds with 32 bytes on the same stream.
+
+An originating peer will only have one outbound stream at a time, and a destination peer
+accepts a maximum of two streams per peer. This is because the destination peer may view
+the originating peer as opening and closing the wrong stream as cross-stream behavior is
+non-linear.
```

Comment:
I don’t understand this paragraph. I also don’t think that we need to talk about resource protection concerns in the documentation, that seems like something that should be discussed in the specs.